samples/net/wifi/shell/sample.yaml

```diff
@@ -68,15 +68,24 @@ tests:
   sample.net.wifi.nrf7002eb:
     extra_args:
       - CONFIG_NRF70_UTIL=y
-      - CONFIG_NRF_WIFI_BUILD_ONLY_MODE=y
+      - CONFIG_WIFI_BUILD_ONLY_MODE=y
       - SHIELD=nrf7002eb
     platform_allow:
       - thingy53/nrf5340/cpuapp
     integration_platforms:
       - thingy53/nrf5340/cpuapp
   sample.net.wifi.nxp_wifi:
     extra_args:
-      - CONFIG_NXP_WIFI_BUILD_ONLY_MODE=y
+      - CONFIG_WIFI_BUILD_ONLY_MODE=y
     platform_allow:
       - frdm_rw612
       - rd_rw612_bga
+  sample.net.wifi.esp32:
+    extra_args:
+      - CONFIG_WIFI_BUILD_ONLY_MODE=y
+    platform_allow:
+      - esp32_devkitc_wroom/esp32/procpu
+      - esp32s2_saola
+      - esp32c3_devkitm
+      - esp32s3_devkitm/esp32s3/procpu
+      - esp8684_devkitm
```

Comment:
consider adding esp32c6 given https://github.com/zephyrproject-rtos/zephyr/pull/81078 might get merged by the time you will be coming back to this PR? :)

Reply:
Sure, will rebase and add C6 accordingly.